### PR TITLE
feat(20087): e2e citizen transaction pages

### PR DESCRIPTION
**Pull request by nnorbert**
e2e citizen transaction pages

### CODE REVIEW

File: apps/payments/e2e/fixtures/transactionsFixtures.ts [hunk 1]
```diff
@@ -38,13 +43,15 @@ export const test = base.extend<transactionFixtures>({
     const referenceCode =
       await manualBankTransferTransactionPage.getReferenceCode();
     const amount = await manualBankTransferTransactionPage.getAmount();
+    const paymentRequestTitle =
+      await manualBankTransferTransactionPage.getPaymentRequestTitle();
```
Comment (peschina) [inline apps/payments/e2e/fixtures/transactionsFixtures.ts:47]: Shouldn't this be the same value of `paymentRequestWithManualBankTransferProvider`?

File: apps/payments/e2e/objects/transactions/CitizenTransactionsPage.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+  async checkHeader() {
+    await expect(this.header).toBeVisible();
+  }
+
+  async goto() {
+    await this.page.goto(`/en/citizen/transactions`);
+  }
+
+  async checkTransaction(transaction: ManualBankTransferTransaction) {
```
Comment (peschina) [inline apps/payments/e2e/objects/transactions/CitizenTransactionsPage.ts:24]: the transaction param here could be any transaction, not necessarily one with manual bank transfer

File: apps/payments/e2e/objects/transactions/CitizenTransactionDetailsPage.ts [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+export class CitizenTransactionDetailsPage {
+  private readonly header: Locator;
+
+  private readonly titleLabel = "Payment Request Title";
+  private readonly amountLabel = "Amount";
+  private readonly lastUpdateLabel = "Last update";
+  private readonly statusLabel = "Status";
+  private readonly providerTypeLabel = "Provider type";
+  private readonly referenceLabel = "Payment reference code	";
```
Comment (peschina) [inline apps/payments/e2e/objects/transactions/CitizenTransactionDetailsPage.ts:12]: Are all this empty spaces intended to be in the label?

File: apps/payments/e2e/specs/citizen/transactionPage.spec.ts [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  test("Should create and list a transaction created by a citizen @smoke @blocker", async ({
+    manualBankTransferTransaction,
+    citizenPage,
+  }) => {
+    await description(
+      "This test checks the listing of transactions created by a citizen",
+    );
+    await owner("OGCIO");
+    await tags("Citizen", "Transaction");
+    await severity(Severity.BLOCKER);
```
Comment (peschina) [inline apps/payments/e2e/specs/citizen/transactionPage.spec.ts:23]: I guess not being able to see the transactions made wouldn't be a blocker? same for the other tests in this suite I guess

**Review by peschina — APPROVED** [review]
LGTM, a few minor comments

File: apps/payments/e2e/fixtures/transactionsFixtures.ts (hunk 1 above)
Comment (nnorbert) [reply apps/payments/e2e/fixtures/transactionsFixtures.ts]: Done!

File: apps/payments/e2e/objects/transactions/CitizenTransactionsPage.ts (hunk 2 above)
Comment (nnorbert) [reply apps/payments/e2e/objects/transactions/CitizenTransactionsPage.ts]: Done!

File: apps/payments/e2e/objects/transactions/CitizenTransactionDetailsPage.ts (hunk 3 above)
Comment (nnorbert) [reply apps/payments/e2e/objects/transactions/CitizenTransactionDetailsPage.ts]: Done!

File: apps/payments/e2e/specs/citizen/transactionPage.spec.ts (hunk 4 above)
Comment (nnorbert) [reply apps/payments/e2e/specs/citizen/transactionPage.spec.ts]: Done!

**Review by peschina — APPROVED** [review]

**Pull request merged**